package.json

```diff
@@ -60,45 +60,47 @@
     "eslint-plugin-react": "^7.5.1",
     "mofo-style": "^2.4.0",
     "optipng-bin": "^4.0.0",
-    "stylelint": "^7.13.0",
+    "stylelint": "^9.2.1",
     "stylelint-config-standard": "^16.0.0",
-    "svgo": "^1.0.3"
+    "svgo": "^1.0.3",
+    "webpack-cli": "^2.1.3"
   },
   "dependencies": {
     "autoprefixer": "^8.3.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-loader": "^7.1.4",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
     "browserslist": "^3.2.6",
     "classnames": "^2.2.5",
     "document-env-vars": "^2.1.0",
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "font-awesome": "^4.7.0",
     "helmet": "^3.9.0",
+    "lodash": "^4.17.10",
     "mofo-bootstrap": "4.0.1",
     "mofo-ui": "^1.7.0",
     "moment": "^2.19.3",
-    "node-sass": "^4.7.2",
+    "node-sass": "^4.9.0",
     "npm-run-all": "^4.1.2",
     "postcss-cli-simple": "^1.0.3",
-    "prop-types": "^15.6.0",
+    "prop-types": "^15.6.1",
     "qs": "^6.5.1",
     "react": "^15.6.2",
-    "react-debounce-input": "^3.1.0",
+    "react-debounce-input": "^3.2.0",
     "react-dom": "^15.6.2",
     "react-formbuilder": "^0.10.0",
-    "react-ga": "^2.3.5",
+    "react-ga": "^2.5.0",
     "react-helmet": "^5.2.0",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
-    "react-select": "^1.1.0",
-    "react-tag-autocomplete": "^5.5.0",
+    "react-select": "^1.2.1",
+    "react-tag-autocomplete": "^5.5.1",
     "shx": "^0.2.1",
-    "snyk": "^1.53.0",
+    "snyk": "^1.80.1",
     "url": "^0.11.0",
-    "webpack": "^2.7.0"
+    "webpack": "^4.8.3"
```

Comment:
This is an impossibly big version update. Let's roll this one back, and file "updating webpack" as a separate issue, because there are breaking changes.

Reply:
:+1: